temoa-uiuc/temoa-uiuc-chws.md

```diff
@@ -0,0 +1,49 @@
+# TEMOA's model of the CHWS
+
+* ExistingCapacity [Done]
+* Efficiency [Done]
+* CostInvest: No Invest Cost because UofI already has it
+* CostFixed: To be added in a future PR
+* CapacityToActivity [Done]
+* CapacityFactorTech [To ask]
+* Storage duration [To ask]
+
+# ExistingCapacity
+
+* 1 ton of chilled water ~ 3.5 kWh [1]
+* COP ~ 5 [2]
+* tons to kg ~ 1000
+* [tons]/(1000 * 5) * 3.5 = [MWe]
+* for example: 52,000 tons yield 36 MW
+* UIUC: 37,500 tons
+* UIUC's total capacity: 6.5e6 millon-gallon ~ 24,600 tons
+* capacity: 37,500 tons
+* time to discharge: 39.4 min
+* electrical-power: 37,500 tons/5e3 * 3.5 = 26.2 MW
+* total electrical-capacity: 26.2 MW * 39.4 min/60 min * 1h = 17.2 MWh
+
+# Efficiency:
+
+* Ideally, we should separate the efficiency of converting electricity into CHW and CHW to electricity.
+* The former is probably slightly lower than the latter.
+* As both are close to 1 anyway, we will consider an efficiency of 1.
+
+# CapacityToActivity:
+
+* Is this the total capacity?
```

Comment:
If a technology of capacity X was operating at 100% for an entire year, how many units of stuff would it produce. 
E.g. 
1 MW (capacity) operating for an entire year will produce 8.76 GWh (units of stuff). 

[Documentation](https://temoacloud.com/temoaproject/Documentation.html#capacity2activity)

Reply:
This information seems redundant. Couldn't Temoa just do: Activity * time ?
I understand that I should multiply the capacity by the number of hours in a year to obtain the Activity - i.e. 37,500 tons x 8760 hours.
However, that Activity is not representative of our system.
If our system can operate at 37,500 tons runs only for 40 min. After that, the system has to recharge.
I don't know how long it takes to recharge, but we could assume that it takes 40 min as well - i.e. we should divide the Capacity by 2. Then, the real capacity would be 37,500 tons x 40min/60min /2 = 12500.
Then, the Activity would be 12500 x 8760 h.

Reply:
And I guess I should have thought this discussion in MW, just consider it equivalent.